doc/cookbook.md

```diff
@@ -30,16 +30,15 @@ While in production, you'll likely want more control than this, this is an easy
 2. Include the following scripts in your page HTML and replace `{your_access_token}` with, well, your access token value and replace `{name_to_use_for_your_component}` with a string identifier that'll be used in the LightStep UI to identify this component:
 
 ```html
-<script src="http://docs.lightstep.com/dist/opentracing-javascript/current/opentracing-browser.min.js"></script>
-<script src="http://docs.lightstep.com/dist/lightstep-tracer-javascript/current/lightstep-tracer.min.js"
+<script src=".../opentracing-browser.min.js"></script>
+<script src=".../lightstep-tracer.min.js"
```

Comment:
The `...` threw me off as it wasn't obviously that wasn't literally what should be there. I might suggest simply leaving it as `<script src="opentracing-browser.min.js"></script>` and leaving it to the reader to set the URL to the context-specific URL.